test/jdk/javax/net/ssl/ServerName/EndingDotHostname.java

```diff
@@ -23,10 +23,11 @@
 
 /**
  * @test
- * @bug 8065422
+ * @bug 8806542 8282511
```

Comment:
No need to add a bug id here as this is test only change

Reply:
Yep.